src/Network/Network/FirewallPolicy/PolicySettings/LogScrubbing/LogScrubbingRule/NewAzureApplicationGatewayFirewallPolicySettingLogScrubbingRuleCommand.cs

```diff
@@ -0,0 +1,74 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Network.Models;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet("New", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "ApplicationGatewayFirewallPolicySettingLogScrubbingRule"), OutputType(typeof(PSApplicationGatewayFirewallPolicySettingLogScrubbingRule))]
+    public class NewAzureApplicationGatewayFirewallPolicySettingLogScrubbingRuleCommand : NetworkBaseCmdlet
+    {
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "State of the log scrubbing config. Default value is Enabled")]
+        [ValidateNotNullOrEmpty]
+        [ValidateSet("Enabled", "Disabled", IgnoreCase = true)]
+        public string State { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The variable to be scrubbed from the logs.")]
+        [ValidateNotNullOrEmpty]
+        [ValidateSet("RequestHeaderNames", "RequestCookieNames", "RequestArgNames", "RequestPostArgNames", "RequestJSONArgNames", "RequestIPAddress", IgnoreCase = true)]
+        public string MatchVariable { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "When matchVariable is a collection, operate on the selector to specify which elements in the collection this rule applies to.")]
+        [ValidateNotNullOrEmpty]
+        [ValidateSet("Equals", "EqualsAny", IgnoreCase = true)]
+        public string SelectorMatchOperator { get; set; }
+
+        [Parameter(
+         Mandatory = false,
+         HelpMessage = "When matchVariable is a collection, operator used to specify which elements in the collection this rule applies to.")]
+        [ValidateNotNullOrEmpty]
```

Comment:
I think in that case user can just not entering the field Selector rather than putting -Selector null as the field is not mandatory.
I will code not having selector to pass * to backend
Should I include null/empty to pass * to backend as well?

Reply:
Mandatory false makes sense. But when it is not passed, we should pass '*' as Selector to backend for EqualsAny. I think NRP already has this code though. Ask Rohit to confirm this. If so, no need to do that, just have EqualsAny operator in your test as well.

Reply:
@tejasshah7 * is passed from the NRP end with Equals Any. Validated it